lib/queuePopulator/IngestionProducer.js

```diff
@@ -171,48 +170,6 @@ class IngestionProducer {
         });
     }
 
-    /**
-     * Get the metadata for each bucket, format and send the info to kafka
-     * so that it can be ingested into MongoDB
-     *
-     * @param {object} bucketList - list of buckets
-     * @param {function} done - callback function
-     * @return {object} list of buckets
-     */
-    _getBucketMd(bucketList, done) {
```

Comment:
On this note, we can probably delete this too: https://github.com/scality/backbeat/blob/f480cbf8c3dcecefecc64c8b4aca46b514c74c11/lib/clients/backbeat-2017-07-01.api.json#L1012

Reply:
*this can be done in a future cleanup/rf pr